### PR TITLE
Citcore: Add cmd's check on adding to avoid duplicated and a cmd proposal to remove all ace for an object

### PR DESCRIPTION
**Pull request by izio38**

### CODE REVIEW

**Comment by izio38** [comment]
Is it normal that server build fail on linux ?

File: code/client/citicore/se/Security.cpp [hunk 1]
```diff
@@ -74,6 +74,11 @@ std::string Context::SaveSnapshot()
 
 void Context::AddPrincipalInheritance(const Principal& child, const Principal& parent)
 {
+	if (DoesPrincipalInheritanceExists(child, parent))
+	{
+		console::Printf("security", "A principal inheritance already exists with this parent and child.\n");
```
Comment (blattersturm) [inline code/client/citicore/se/Security.cpp:79]: Warning on this is a little odd. :(

Comment (izio38) [reply code/client/citicore/se/Security.cpp:79]: Should we completely remove this ?

**Comment by izio38** [comment]
Still waiting for a review :)

**Pull request closed without merging**